flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -103,6 +106,23 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 */
 	Optional<Catalog> getCatalog(String catalogName);
 
+	/**
+	 * Load a {@link Module} under a unique name.
```

Comment:
mention that the module will be added to the end of list?

Reply:
mentioning "list" may be revealing too much implementation details on API. I'll rephrase the comment here